### DISCUSSION TITLE
23 marzo 2023 - esercizio 2

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#36] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/f1443acb-4f1f-471f-9344-344f661cf3cc)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/8b689e88-2e88-461b-9ab2-35582bd47769)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/a75a2e47-a943-45f9-a423-c0907cb2106a)

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-15 14:43:15](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/42181162-4af0-4fc0-a7fb-e7d1ecefd765)
![IMAGE 2024-01-15 14:43:20](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/7af33aeb-c5b9-4907-b021-6a1f377bf715)

**Comment by daromacarlo** [comment]
![Immagine 2024-02-04 190536](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/137cca4d-9ded-4b74-aed5-b8996c370951)
![Immagine 2024-02-04 190551](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/27e8a43b-56b6-43eb-bff8-a56db09f11aa)
![Immagine 2024-02-04 190603](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/8237b97d-e437-4a62-af7f-da2cc6149ea7)
![Immagine 2024-02-04 190614](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/5a5021c9-231e-474d-9c1e-f661e0aecb7f)
![Immagine 2024-02-04 190631](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/3543c22b-232b-464e-b7cf-9435d64fb142)

**Reply by salicius** [reply]
Nell'es 2b: Perché il calcolo di B->AC lo metti nella seconda iterazione e non direttamente nella prima ?

**Reply by daromacarlo** [reply]
@salicius scusami ma il mio cervello ha completamente rimosso questa materia dopo l'esame... Ti conviene proporre la domanda sul gruppo Whatsapp.
